src/pages/[lang]/services/index.astro

```diff
@@ -43,12 +46,35 @@ const t = useTranslations(lang, 'services');
 
 	<div class="motion-safe:animate-fade-up motion-safe:animate-delay-500 relative flex flex-col mb-20 md:mb-40 xl:flex-row py-20 before:-z-10 before:bg-light-blue-50 before:absolute before:-left-[3000px] before:top-0 before:bottom-0 before:right-[1000px] marker:before:h-full before:w-[10000px] before:rounded-xl">
 		<div class="motion-safe:animate-fade-up motion-safe:animate-delay-700 flex flex-col items-start w-full md:pr-12 gap-2 mb-8 md:mb-16 xl:sticky xl:w-4/12 h-fit top-40">
-		  <h2 class="mb-2 font-serif text-2xl text-pretty" set:html={t('overview.headline')}></h2>
+		  <h2 class="mb- font-serif text-2xl text-pretty" set:html={t('overview.headline')}></h2>
+		  <dl class="">
```

Comment:
Sure you wanted to use a <dl>, a dl can't have <li>, either a <div>, or <dt> and <dd> pairs